tutorials/introductory/sample_plots.py

```diff
@@ -54,7 +88,7 @@
 
 .. _screenshots_path_demo:
 
-Path demo
+Paths
 =========
```

Comment:
I agree that the current version is valid RST, but would it be possible to match the length of the '====' line with the line beyond?